op/model.py

```diff
@@ -269,6 +298,11 @@ def _run(self, *args):
         data = json.loads(text)
         return data
 
+    def _run_text(self, *args):
```

Comment:
How about having a json=True/False argument on _run itself?  Otherwise we'll end up with lots of small variations and any error handling we do will need to be split out into another common function anyway.